src/ext/configuration.h

```diff
@@ -30,8 +31,22 @@ bool ddtrace_config_env_bool(ddtrace_string env_name, bool default_value TSRMLS_
 bool ddtrace_config_distributed_tracing_enabled(TSRMLS_D);
 bool ddtrace_config_trace_enabled(TSRMLS_D);
 
+#define DDTRACE_LONGEST_INTEGRATION_ENV_PREFIX_LEN 10  // "DD_TRACE_" FTW!
+#define DDTRACE_LONGEST_INTEGRATION_ENV_SUFFIX_LEN 23  // "_ANALYTICS_SAMPLE_RATE" FTW!
```

Comment:
These are one larger than the string lengths; any reason for that? Usually +1 is for NULL but since we're concatenating strings there won't be nulls between them.

Reply:
Nope, that's just an off by one error from entering in what my IDE was telling me. 

<img width="422" alt="Screen Shot 2020-07-29 at 6 56 08 AM" src="https://user-images.githubusercontent.com/578780/88810043-aea45d00-d169-11ea-9842-1d8d17c2fa2f.png">

Nice catch!